README.md

```diff
@@ -57,3 +54,6 @@ To publish your changes, you'll need to increment the version number in `./versi
                 * ( done ) print a warning
                 * ( done ) have the build send an email (set build to unstable)
 * ( done ) add a "clean" script that removes tmp, maybe clears out some docker images, too?
+* ( TODO ) does/can/should ci/test work when not run in the image builder container?
+* ( TODO ) automatically publish git tags for published versions
+* ( TODO ) publish version 1.0.
```

Comment:
We should coordinate that with whatever release of the portal we decide to put this in.

Reply:
I was thinking in terms of semver that when we are ready to use this (about now), the major version I think should technically be at least 1 because otherwise it suggests alpha/beta. It's easy to do any time we are ready.